.github/workflows/flowzone.yml

```diff
@@ -991,40 +897,71 @@ jobs:
         uses: docker/metadata-action@v4
         with:
           images: |
-            ${{ env.DOCKER_IMAGES }}
+            ${{ env.DOCKER_IMAGES || env.SUT_IMAGE }}
```

Comment:
The previous merge was missing this fallback for when the images list is empty. We still want to build and push to the local docker registry for testing.